tests/testthat/test-DataDiscoveryRetrieval.R

```diff
@@ -1,3 +1,114 @@
 test_that("multiplication works", {
   expect_equal(2 * 2, 4)
 })
+
+test_that("TADAdataRetrieval ", {
+  tada1 <- TADAdataRetrieval(siteid = c("USGS-054064785",
+                                        "USGS-430305089260600"),
+                             characteristicName = "Phosphorus")
+  # you could just pick the important columns:
+  expect_true(all(c("OrganizationIdentifier",                           
```

Comment:
You can make a lot more tests on the output here, I just am checking that all the columns we expect are there.